### PR TITLE
Add runipy

### PR DESCRIPTION
**Pull request by jakirkham**
Simply generated the recipe using the wonderful `conda skeleton pypi`. Made a few tweaks to make it fit nicely in conda-forge.

### CODE REVIEW

**Comment by jakirkham** [comment]
Looks like we need to be setting unicode support in the docker container. Though I can probably also lose whatever unicode character crept into the readme too.

cc @jjhelmus

**Comment by jakirkham** [comment]
Borrowing @ocefpaf commit ( https://github.com/conda-forge/staged-recipes/pull/150/commits/4973ec855e5b7d833286f5c747ddf3db7950ac86 ). Seems to fix our issues with unicode. May want to check that we do have `locales` installed and configured though.

**Comment by jakirkham** [comment]
Checks out (given the UTF8 change).

**Comment by jakirkham** [comment]
Thanks @ocefpaf.

**Comment by jakirkham** [comment]
@paulgb, if you are interested, I can add you as a maintainer to the `runipy` conda package.

File: scripts/run_docker_build.sh [hunk 1]
```diff
@@ -40,7 +40,7 @@ conda install --yes anaconda-client obvious-ci
 conda install --yes conda-build-all
 
 conda info
-unset LANG
+export LANG=en_US.UTF-8
```
Comment (pelson) [inline scripts/run_docker_build.sh]: This has changed the global staged-recipes setup, not just that for runipy. I don't know if that is healthy or not, but I must have put the unset there for a reason... either way, we need a comment adding to this. @jakirkham - would you mind opening a PR which adds such a comment?

**Comment by pelson** [comment]
:+1: on the recipe. Just one small question on the change to the staged-recipes Linux image.

File: scripts/run_docker_build.sh (hunk 1 above)
Comment (jakirkham) [inline scripts/run_docker_build.sh]: Definitely. Added a PR ( https://github.com/conda-forge/staged-recipes/pull/157 ).

**Pull request merged**